### PR TITLE
Japanese Translated 2021-Sep

### PR DESCRIPTION
**Pull request by konabuta**
# PR into Azure/azureml-cheatsheets

## Checklist

I have:

- [x] read and followed the [contributing guidelines](https://azure.github.io/azureml-cheatsheets/docs/misc/contributing)

## Changes

- Translated several markdown.
- Change submenu into Japanese.

fixes #
- Link name to Japanese

### CODE REVIEW

**Comment by konabuta** [comment]
@fuhuifang  @aminsaied Could you please review this pull request? Thanks.

File: website/docs/cheatsheets/python/v1/distributed-training.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ Simply ensure that you have taken care of the following:
 * Create an `MpiConfiguration` with your desired distribution.
 
 #### Example
-* [azureml-examples: TensorFlow distributed training using Horovod](https://github.com/Azure/azureml-examples/tree/main/python-sdk/workflows/train/tensorflow/mnist-distributed-horovod)
```
Comment (fuhuifang) [inline website/docs/cheatsheets/python/v1/distributed-training.md:69]: Hi @konabuta, links on this page are broken. and I've fixed them in #79 . Could you pull the latest version from main branch?

**Review by fuhuifang — CHANGES_REQUESTED** [review]
Hi @konabuta, links on this page are broken. and I've fixed them in #79  . Could you pull the latest version from main branch?

**Comment by konabuta** [comment]
@fuhuifang I changed the link according to #79. I found [azureml-examples: Multi-node training with PyTorch Lightning](https://github.com/Azure/azureml-examples/blob/main/python-sdk/experimental/using-pytorch-lightning/4.train-multi-node-ddp.ipynb) is deleted, so I commented out.

File: website/docs/cheatsheets/python/v1/distributed-training.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ Simply ensure that you have taken care of the following:
...
+* [azureml-examples: TensorFlow distributed training using Horovod](https://github.com/Azure/azureml-examples/tree/main/workflows/train/tensorflow/mnist-distributed-horovod)
```
Comment (fuhuifang) [inline website/docs/cheatsheets/python/v1/distributed-training.md:69]: Hi @konabuta, could you change the links on this page too? They should be already updated in #79.

**Review by fuhuifang — CHANGES_REQUESTED** [review]

File: website/docs/cheatsheets/python/v1/distributed-training.md (hunk 2 above)
Comment (konabuta) [reply website/docs/cheatsheets/python/v1/distributed-training.md]: Sorry I missed this file. I modified that points.

**Review by fuhuifang — APPROVED** [review]

**Comment by konabuta** [comment]
@fuhuifang  Thank you for your review.

**Pull request merged**